internal/scout/usage/docker.go

```diff
@@ -2,27 +2,88 @@ package usage
 
 import (
 	"context"
+	"encoding/json"
 	"fmt"
 
+	"github.com/charmbracelet/bubbles/table"
+	"github.com/docker/docker/api/types"
 	"github.com/docker/docker/client"
+	"github.com/sourcegraph/sourcegraph/lib/errors"
+	"github.com/sourcegraph/src-cli/internal/scout/style"
 )
 
 func Docker(ctx context.Context, client client.Client, opts ...Option) error {
 	cfg := &Config{
-		namespace:     "default",
-		docker:        true,
-		pod:           "",
-		container:     "",
-		spy:           false,
-		k8sClient:     nil,
-		dockerClient:  &client,
-		metricsClient: nil,
+		namespace:    "default",
+		docker:       true,
+		pod:          "",
+		container:    "",
+		spy:          false,
+		dockerClient: &client,
 	}
 
 	for _, opt := range opts {
 		opt(cfg)
 	}
 
-	fmt.Println("Command is routed, but not yet fixed.")
+	containers, err := cfg.dockerClient.ContainerList(ctx, types.ContainerListOptions{})
+	if err != nil {
+		return errors.Wrap(err, "could not get list of containers")
+	}
+
+	return renderDockerUsageTable(ctx, cfg, containers)
+}
+
+// renderDockerUsageTable renders a table displaying CPU and memory usage for Docker containers.
+func renderDockerUsageTable(ctx context.Context, cfg *Config, containers []types.Container) error {
+	columns := []table.Column{
+		{Title: "Container", Width: 20},
+		{Title: "Cores", Width: 10},
+		{Title: "Usage", Width: 10},
+		{Title: "Memory", Width: 10},
+		{Title: "Usage", Width: 10},
+	}
+	rows := []table.Row{}
+
+	for _, container := range containers {
+		containerInfo, err := cfg.dockerClient.ContainerInspect(ctx, container.ID)
+		if err != nil {
+			return errors.Wrap(err, "failed to get container info")
+		}
+
+		stats, err := cfg.dockerClient.ContainerStats(context.Background(), container.ID, false)
+		if err != nil {
+			return errors.Wrap(err, "could not get container stats")
+		}
+		defer func() {
+			_ = stats.Body.Close()
+		}()
```

Comment:
Go style nit: this should be equivalent
```suggestion
		defer stats.Body.Close()
```